custom/aaa/models.py

```diff
@@ -691,14 +691,14 @@ def agg_from_growth_monitoring_forms_ucr(cls, domain, window_start, window_end):
                    array_agg(zscore_grading_wfa) AS zscore_grading_wfa_history
```

Comment:
Do these add the new values to an existing array

Reply:
the subquery returns rows like:

child_health_1, [time1, 5], [time1, 70cm] ...
child_health_1, [time2, 6], [time2, 71cm] ...
child_health_1, [time3, 7], [time3, 72cm] ...


This part of the query groups by child_health_case_id and transforms that to

child_health_1, [[time1,5],[time2,6][time3,7]], [[time1,70cm], ....

So that each row has the entire history at once